client/signup/config/flows.js

```diff
@@ -84,6 +84,13 @@ const flows = {
 		lastModified: '2016-01-27'
 	},
 
+	subdomain: {
+		steps: [ 'design-type', 'themes', 'domains', 'plans', 'survey-user' ],
+		destination: getSiteDestination,
+		description: 'Provide a vertical for subdomains',
```

Comment:
I don't think so, I believe this is just for internal reference so that we know what each flow was added for. Same as the date. cc @michaeldcain